R/utils.R

```diff
@@ -86,3 +86,10 @@ name_dots <- function(...) {
   list(vnames=vnames, novname=novname)
 }
 
+# convert a vector like c(1, 4, 3, 2) into a string like [1, 4, 3, 2]
+#   (common aggregation method for error messages)
+brackify = function(x) {
+  # arbitrary cutoff
+  if (length(x) > 10L) x = c(x[1:10], '...')
```

Comment:
This is very nice you handle that already but could also be tested

Reply:
@jangorecki see #3117